minor rewording of power sign documentation

rivengh opened the pull request.

rivengh committed:
minor rewording of power sign documentation

andig commented:
Bin nicht sicher obs das besser macht. Consumption ist z.b. der Fachausdruck siw.

rivengh commented:
Ok, mir ging es primär um feed-in für Netzeinspeisung. Das könnte vielleicht etwas verwirren.

premultiply commented:
Im Netzwerkbereich nutzt man ingress und egress oder inbound und outbound.

andig commented:
Das ist aber ganz schön offtopic, hier ist ja Energiewirtschaft der Kontext. M.e. Sind consumption und feed-in richtig?

andig commented:
@rivengh physikalisch tue ich mich auch mit incoming/outgoing Power schwer. Wenn etwas rein/raus kommt dann egtl. die Energie, Leistung ist ja nur das Maß für die "Intensität"?

premultiply commented:
Also die Brexit-Insulaner sprechen zumindest von import und export.
Siehe z. B. hier: https://www.theenergydesk.co.uk/ted-green/green-connections/import-export-electricity-meters/

rivengh commented:
> Im Netzwerkbereich nutzt man ingress und egress oder inbound und outbound.
Würde ich in diesem Kontext hier auch eher nicht verwenden.

Import/Export bzw. Incoming/Outgoing passt hier m.E. besser. Aus Sicht vom EVCC tendiere ich zu letzterem, da ich persönlich Import/Export eher mit Grid/Hausanschluss verbinde. Aber beides OK, sollte dann bei einem Wording bleiben.

> @rivengh physikalisch tue ich mich auch mit incoming/outgoing Power schwer. Wenn etwas rein/raus kommt dann egtl. die Energie, Leistung ist ja nur das Maß für die "Intensität"?

Stimmt, Energy ist hier besser. Ich hatte es hier korrigiert da einmal Power und einmal Energy verwendet wurde.

rivengh committed:
Update README.md

Used energy instead of power and grid consumption instead of grid purchases in power sign documentation.

andig commented:
So finde ichs super, danke!
